<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorationEditorClass</class>
 <widget class="QDialog" name="ColorationEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ColorationEditor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lEdtName"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pBtnColor">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pBtnValid">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="DebuggerUI.qrc">
       <normaloff>:/DebuggerUI/resources/valid.png</normaloff>:/DebuggerUI/resources/valid.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pBtnRemove">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="DebuggerUI.qrc">
       <normaloff>:/DebuggerUI/resources/remove.png</normaloff>:/DebuggerUI/resources/remove.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QListWidget" name="lwdgColor">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DebuggerUI.qrc"/>
 </resources>
 <connections/>
</ui>
